.gitignore

```diff
@@ -1,5 +1,24 @@
+# Composer Dependencies
 /vendor/*
+/bin
```

Comment:
I'm confused over that actually trying to read between the two projects why they exist in one but not the other.